backend/homework/7/src/main/java/org/example/Factory1VehicleService.java

```diff
@@ -0,0 +1,36 @@
+// Factory1VehicleService.java
+package org.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Factory1VehicleService {
+
+    private ServiceTyre tyreService;
+    private ServiceSpeaker speakerService;
+
+    @Autowired
+    public Factory1VehicleService(
+            @Qualifier("tyre1") ServiceTyre tyreService,
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.